File: app/src/main/java/com/androidstudy/mpesa/di/ActivitiesModule.java
package com.androidstudy.mpesa.di;

import com.androidstudy.mpesa.ui.PaymentActivity;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
abstract class ActivitiesModule {

    @ContributesAndroidInjector
    abstract PaymentActivity contributesPaymentActivity();


}

File: app/src/main/java/com/androidstudy/mpesa/di/ActivitiesModule.kt
package com.androidstudy.mpesa.di

import com.androidstudy.mpesa.ui.PaymentActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
internal abstract class ActivitiesModule {
    @ContributesAndroidInjector
    abstract fun contributesPaymentActivity(): PaymentActivity?
}